plugins/image-prioritizer/class-ip-img-tag-visitor.php

```diff
@@ -68,12 +66,26 @@ public function __invoke( OD_HTML_Tag_Walker $walker ): bool {
 			 * Note also that if this is the LCP element for _some_ of the viewport groups, it will still get
 			 * fetchpriority=high by means of the preload link (with a media query) that is added further below.
 			 */
-			$walker->set_attribute( 'data-od-removed-fetchpriority', $walker->get_attribute( 'fetchpriority' ) );
 			$walker->remove_attribute( 'fetchpriority' );
 		}
 
-		// TODO: If the image is visible (intersectionRatio!=0) in any of the URL metrics, remove loading=lazy.
-		// TODO: Conversely, if an image is the LCP element for one breakpoint but not another, add loading=lazy. This won't hurt performance since the image is being preloaded.
+		// TODO: Also if the element isLCPCandidate it should never by lazy-loaded.
```

Comment:
But in this case, clearly the `intersectionRatio` will be non-zero. So this is redundant.